kodeagent.py

```
@@ -331,10 +336,55 @@ def search_wikipedia(query: str, max_results: Optional[int] = 3) -> str:
+            query=query,
+            max_results=max_results,
+            sort_by=arxiv.SortCriterion.Relevance
+        )
+
+        results = list(client.results(search))
+
+        if not results:
+            return f'No results found for the query: {query}'
+
+        output = f'## ArXiv Search Results for: {query}\n\n'
+        for i, result in enumerate(results):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)